openhands/runtime/base.py

```diff
@@ -47,22 +48,6 @@
 }
 
 
-class RuntimeUnavailableError(Exception):
-    pass
-
-
-class RuntimeNotReadyError(RuntimeUnavailableError):
-    pass
-
-
-class RuntimeDisconnectedError(RuntimeUnavailableError):
```

Comment:
I think these changes are merged elsewhere?

Reply:
yep - i need to get this PR updated!